rfcs/0114-Abuse protection for RabbitMQ server and tc-events.md

```diff
@@ -0,0 +1,28 @@
+# RFC 114 - Abuse protection for RabbitMQ server and tc-events.md
+* Comments: [#114](https://api.github.com/repos/taskcluster/taskcluster-rfcs/issues/114)
+* Proposed by: @Biboswan
+
+# Summary
+
+1: limit connection attempt rate to tc-events(frequent opening and closing connection => higher CPU utilization)
+
+2: Use maxLength, Prefetch properties for queues (Memory limit)
+
+3:Using memory and cpu alarm apis (https://www.cloudamqp.com/docs/api-alarms.html)
+Our rabbitmq server is accessed by other ways as well.Hence this will give us a better understanding of the rabbitmq server for tc-events to act accordingly.
+
+4: Disable lazy queues. Yes if incoming message rate is quite high in comparison to consumption rate then better to enable lazy queues. Just some thoughts if some optimization mid way possible
```

Comment:
All of the queues managed by this service are non-durable queues, so IMHO they should not be writing to disk at all, and ideally their size should be severely limited.